steam_wrapper/steam_wrapper.py

```diff
@@ -246,6 +246,26 @@ def configure_shared_library_guard():
         if mode > 1:
             os.environ["LD_BIND_NOW"] = "1"
 
+def setup_proton_extensions():
+    # Create root symlink if it doesn't exist
+    src = STEAM_ROOT + '/.local/share/Steam'
+    dst = STEAM_ROOT + '/.steam/root'
+    proton_dest = src + '/compatibilitytools.d'
+
+    if not os.path.isdir(os.path.dirname(dst)):
+        os.makedirs(os.path.dirname(dst))
+    if not os.path.isdir(os.path.dirname(src)):
+        os.symlink(src, dst)
+
+    # Try to create folder if it doesn't exist
+    os.makedirs(proton_dest, exist_ok=True)
+
+    # Copy extensions if they exist
+    subfolders = [f.path for f in os.scandir("/app/proton") if f.is_dir() ]
+
+    for proton in subfolders:
+        shutil.copy(proton, proton_dest)
```

Comment:
Removing stale content is probably not something we want to do since wine is bound to break certain games after updates so it's still nice to be able to let them keep the old versions.